### PR TITLE
Close secondary action menu when clicking action

### PR DESCRIPTION
**Pull request by johnhwhite**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-action-bars?_sp.spa=pr-close-menu-when-clicking-action](https://developer.blackbaud.com/skyux-action-bars?_sp.spa=pr-close-menu-when-clicking-action)

File: src/app/public/modules/summary-action-bar/actions/summary-action-bar-secondary-actions.component.ts [hunk 1]
```diff
@@ -74,6 +82,9 @@ export class SkySummaryActionBarSecondaryActionsComponent implements AfterConten
       }
       this.secondaryActionComponents.forEach(action => {
         action.isDropdown = isDropdown;
+        action.actionClick.subscribe(() => {
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/modules/summary-action-bar/actions/summary-action-bar-secondary-actions.component.ts:85]: Should we unsubscribe to this subscription on destroy?

**Review by Blackbaud-AlexKingman — COMMENTED** [review]
One minor comment. Also, should we have a unit test for this?

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-action-bars?_sp.spa=pr-close-menu-when-clicking-action](https://developer.blackbaud.com/skyux-action-bars?_sp.spa=pr-close-menu-when-clicking-action)

**Review by Blackbaud-AlexKingman — APPROVED** [review]

**Pull request merged**